compose.yaml

```diff
@@ -56,7 +62,7 @@ services:
 
   websocket:
     <<: *depends_on_configurator
-    image: frappe/erpnext:${ERPNEXT_VERSION}
+    image: frappe/erpnext:${ERPNEXT_VERSION:?No ERPNext version set}
```

Comment:
websocket image can also be custom. it'll have Frappe and socketio

Reply:
@maovidal can you also change this image? No need to keep it. Custom image will have socketio in it.

Reply:
Done.
Hey @revant, maybe not the place nor the time, but certainly I want to express my gratitude to you for all you have shared on this repo. I've learned a lot.